test/ui/LoginTest.ts

```diff
@@ -0,0 +1,186 @@
+import { expect } from "chai";
+import { LoginPage } from "./LoginPage";
+
+
+describe("LoginPage Tests", () => {
+  let loginPage: LoginPage;
+
+  before(async () => {
+    loginPage = new LoginPage();
+    await loginPage.open();
+  });
+
+  after(async () => {
+    await loginPage.closeBrowser();
+  });
+
+  // vars
+  let adminValidEmail: string = "valid.admin@email.com";
```

Comment:
Probably need to get rid of hard coded values here. Again we can add these as secrets and call them if necessary.